chill-algebird/src/main/scala/com/twitter/chill/algebird/AlgebirdSerializers.scala

```diff
@@ -95,3 +86,42 @@ class HLLMonoidSerializer extends KSerializer[HyperLogLogMonoid] {
     hllMonoids.getOrElseUpdate(bits, new HyperLogLogMonoid(bits))
   }
 }
+
+class SketchMapSerializer[K, V](implicit skmMonoid: SketchMapMonoid[K, V], valueMonoid: Monoid[V])
+  extends KSerializer[SketchMap[K, V]] {
+
+  def write(kryo: Kryo, output: Output, skm: SketchMap[K, V]) {
+    val rows = skm.valuesTable.rows
+    val cols = skm.valuesTable.cols
+
+    val values: IndexedSeq[(Int, Int, V)] =
+      for (
+        r: Int <- 0 to rows - 1; c: Int <- 0 to cols - 1 if skm.valuesTable.getValue((r, c)) != valueMonoid.zero
+      ) yield (r, c, skm.valuesTable.getValue((r, c)))
+
+    output.writeInt(rows, true)
+    output.writeInt(cols, true)
+    kryo.writeClassAndObject(output, values.toList)
```

Comment:
4. Also, I think you're better off writing `((Int, Int), V)` than `(Int, Int V)` because tuple2 is specialized and it won't write the type token twice.